proteus/LinearSolvers.py

```diff
@@ -581,10 +610,309 @@ def failed(self):
     def info(self):
         self.ksp.view()
 
-class NavierStokes3D:
+class schurOperatorConstructor:
```

Comment:
We go  by the convention that derived  types (class names) are usually capitalized while object names are lower case.